components/RefinementList.js

```diff
@@ -9,10 +9,12 @@ class RefinementList extends React.Component {
 
   _generateFacetItem(facetValue) {
     var hasChildren = facetValue.data && facetValue.data.length > 0;
+    var subList = hasChildren && <RefinementList {...this.props} facetValues={facetValue.data} />;
+    var data = facetValue;
 
-    var subList = hasChildren ?
-      <RefinementList {...this.props} facetValues={facetValue.data} /> :
-      null;
```

Comment:
React advise people to use null for an empty component, that's not the same as false. Cannot find the reference again but I think there's a difference, we'll see. (This was uneeded refactoring here right?)